app/presenters/migration/new.rb

```diff
@@ -15,6 +15,112 @@ def page_content_heading
       'Tell your story'
     end
 
+    def fields
+      {
+        items: [
```

Comment:
Is it necessary to have `fields` be a Hash with one element? Could we not just have `fields` return the Array of fields?

Reply:
I had it in mind to add in a "legend" key under fields for grouping them, hence the "items" being the array.  This structure also leaves a space for a field to specify if a group should be shown as part of the basic or advanced form. 

Also with the mega-form in mind we could allow "items" to contain further "field" structures.